addons/message_update_v2.15/manifest.json

```diff
@@ -22,18 +22,14 @@
         "content": [
           {
             "id": "l_1",
-            "content": "You can now choose to filter out malware, ads, and ad trackers, giving you more protection as you navigate the web. Find these features in “Privacy features” within the Settings menu."
+            "content": "We will now recommend a few of the highest-performing server locations when you’re on the Select Location screen."
           },
           {
             "id": "l_2",
-            "content": "When you first launch the app, the app will now automatically suggest the server location that is closest to you."
+            "content": "We’ve updated the App Exclusions screen to make it easier to choose which apps to leave out of VPN protection."
```

Comment:
Should have made it clear that this needed fixing also here.

Reply:
Sheesh I am sorry flod, I should have paid more attention. Will send the fix in a separate PR.